src/App.tsx

```diff
@@ -1,7 +1,36 @@
-import React from 'react';
+import React, { useState } from 'react';
+import cn from 'classnames';
 import 'bulma/css/bulma.css';
 import './App.scss';
 
+type SortParams = {
+  property: string,
+  reverse: boolean,
+};
+
+function getGoods(goods: string[], sortParams: SortParams) {
+  const goodsCopy = [...goods];
+
+  if (sortParams.property) {
+    goodsCopy.sort((good1, good2) => {
+      switch (sortParams.property) {
+        case 'name':
```

Comment:
please move 'name' and 'length' to the `SortType` enum as it will make a code cleaner

Reply:
Not fixed from the previous review